packages/forklift-console-plugin/src/components/page/StandardPageWithExpansion.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react';
```

Comment:
this is basically a copy of packages/forklift-console-plugin/src/components/page/StandardPageWithSelection.tsx but without selection, only expansion